com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -485,10 +635,38 @@ private void SendAllParamsAndState()
             }
         }
 
+        private ulong[] GetTargetClientIds(ulong originClientId)
```

Comment:
At a very least, would like this function to live only in ClientNetworkAnimator, because as I said, that's where the client-ish functionality needs to live, and this is only needed for CNA.

And, more like NetworkTransform, I'd prefer were your CNA just ignores what comes its way rather than go to this bookkeeping trouble. After all, it should have its own local state, and it knows that it's its owner, so it can just skip that and avoid all this bookkeeping, no?

Reply:
This bookkeeping allows to use slightly less bandwidth by avoiding sending the data that the authoritative client already has.

Reply:
I hear you, but 1) I'm not sure it's worth the code complexity, and 2) I'd rather this bubble up to doing this properly in the RPC system as it would seem a not uncommon need to say "send to everyone, I don't need my own RPC echoed to me".

Reply:
makes sense, it is indeed better to optimize at an appropriate layer